greenpithumb/db_store.py

```diff
@@ -26,7 +26,7 @@ def store_soil_moisture(self, timestamp, soil_moisture):
             soil_moisture: An int of the soil moisture reading.
         """
         self._cursor.execute('INSERT INTO soil_moisture VALUES (?, ?)',
-                             (timestamp, soil_moisture))
+                             (str(timestamp), soil_moisture))
```

Comment:
Can we create a function like `_serialize_timestamp` that takes a `datetime` and returns a string? That way all the functions share a single implementation and we can change the format easily in one spot.

I think we should do `timestamp.isoformat` because it's more obvious how it's serializing it. I wouldn't know off the top of my head that `str(timestamp)` serializes it in ISO format unless I jumped to the documentation or tests.